src/main/java/bio/terra/common/exception/SamApiException.java

```diff
@@ -34,7 +38,11 @@ public static SamApiException create(ApiException apiException) {
         logger.debug("Unable to deserialize sam exception response body");
       }
     }
-    return new SamApiException(message, apiException);
+    if (!messagePrefix.isEmpty()) {
```

Comment:
nit: if we do StringUtils.isEmpty, we'll handle the null case.

Reply:
done